tests/src/Functional/ContentEventRenderTest.php

```diff
@@ -407,7 +407,7 @@ public function testEventRendering(): void {
           'body' => "Friday 28 February 2020, 01:00\n - Monday 9 March 2020, 01:00",
         ], [
           'label' => 'Livestream',
-          'body' => 'Starts on Wednesday 18 March 2020, 01:00',
+          'body' => 'Starts on Wednesday 18 March 2020, 01:00Link to online event',
```

Comment:
I've raised the question above - why do we show the link to the event. There is a comment
`    // The livestream is upcoming so only the start date is available.`

Reply:
yes. fixed.